cmake/InstallHelpers.cmake

```diff
@@ -107,11 +107,11 @@ function(install_helper)
 
   if(_install_helper_TARGETS)
     # Ensure the bin area exists with the correct permissions.
-    create_install_dir_with_perms(DESTINATION bin)
+    create_install_dir_with_perms(DESTINATION "${CMAKE_INSTALL_BINDIR}")
```

Comment:
Does CMake have a notion of inheriting includes? This worked fine, so I assume something of that nature is happening.